.pre-commit-config.yaml

```diff
@@ -11,6 +11,13 @@
 # These pre-commit hooks are run as CI.
 #
 # NOTE: if it can be avoided, add configs/args in pyproject.toml, setup.cfg or below instead of creating a new `.config.file`.
+# https://pre-commit.ci/#configuration
+ci:
+  autoupdate_schedule: monthly
+  autofix_commit_msg: |
+    [pre-commit.ci] Apply automatic pre-commit fixes
+  # this does not work on pre-commit ci
+  skip: [terraform_fmt]
```

Comment:
I pinged you about this in #1545 👍

Reply:
I just saw that - I added the pre-commit.ci so that we do not have to manually update/upgrade hooks but the terraform one does not seem to work on there so I added the workaround here